### DISCUSSION TITLE
"See similar reports"

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#15] [New Feature Requests]
As a user, I would like to see similar reports from the one I am reviewing.

For example, suppose I am studying lending protocols and I see an issue explaining how a borrower can be liquidated even if they have a good collateralization ratio. Now I want to explore other findings from the same category. It would be interesting if there was a list of other reports that contain the same type of problem.

This could be achieved by performing some sort of clustering based on the content of the report, possibly using ChatGPT, but maybe not necessarily, as other ML algos can do that well I suppose. 

Btw: if you're intending on having a "Pro" plan, I would pay for this feature

**Comment by hans-solodit** [comment]
@aviggiano 
Actually this feature is already there. Probably we need to run our GPT tasks for the latest reports.
Also, the reports in PDF format are not included in the GPT tasks.
Maybe we would think a way how to show the similar ones so that users can find more easily.

By the way, we are truly committed providing a completely free platform!


![screenshot_07](https://github.com/solodit/feedback/assets/129466917/4c74dbff-10fb-49c1-a05f-d8f8975040a9)

**Reply by aviggiano** [reply]
Amazing! Yes the report I was reading was a PDF, that's probably why I didn't see it.

Thanks, awesome work
